### PR TITLE
Update command-line-arguments.md

### PR DESCRIPTION
**Pull request by Youssef1313**

### CODE REVIEW

File: docs/csharp/programming-guide/main-and-command-args/command-line-arguments.md [hunk 1]
```diff
@@ -2,75 +2,77 @@
...
+
+For more information, see <xref:System.Int64.Parse%2A> and <xref:System.Convert>.
+
+## Example
+
+The following example shows how to use command-line arguments in a console application. The application takes one argument at run time, converts the argument to an integer, and calculates the factorial of the number. If no arguments are supplied, the application issues a message that explains the correct usage of the program.
+
+To compile and run the application from a command prompt, follow these steps:
+
+1. Paste the following code into any text editor, and then save the file as  a text file with the name *Factorial.cs*.
```
Comment (BillWagner) [inline docs/csharp/programming-guide/main-and-command-args/command-line-arguments.md]: This change should be reverted.

The docs style guide calls for filenames to be *italic*. (The site engineering team is working on features so they won't be localized.)

File: docs/csharp/programming-guide/main-and-command-args/command-line-arguments.md [hunk 2]
```diff
@@ -2,75 +2,77 @@
...
+     [!code-csharp[csProgGuideMain#16](~/samples/snippets/csharp/VS_Snippets_VBCSharp/csProgGuideMain/CS/Class1.cs#16)]
+
+2. From the **Start** screen or **Start** menu, open a Visual Studio **Developer Command Prompt** window, and then navigate to the folder that contains the file that you just created.
+
+3. Enter the following command to compile the application.
   
      `csc Factorial.cs`  
   
-     If your application has no compilation errors, an executable file that's named `Factorial.exe` is created.  
+     If your application has no compilation errors, an executable file that's named *Factorial.exe* is created.
```
Comment (BillWagner) [inline docs/csharp/programming-guide/main-and-command-args/command-line-arguments.md]: same as above.

**Review by BillWagner — CHANGES_REQUESTED** [review]
This is great @Youssef1313 

I did have a couple comments based on the style guide. Once you update those, this is ready to merge.

File: docs/csharp/programming-guide/main-and-command-args/command-line-arguments.md (hunk 1 above)
Comment (Youssef1313) [reply docs/csharp/programming-guide/main-and-command-args/command-line-arguments.md]: The change follows the style guide, it made Factorial.cs italic. I didn't get why it should be reverted ?

Comment (BillWagner) [reply docs/csharp/programming-guide/main-and-command-args/command-line-arguments.md]: My mistake. I'll :shipit: now.

**Review by BillWagner — APPROVED** [review]

**Pull request merged**